lib/features/family/features/bedtime/presentation/pages/setup_bedtime_screen.dart

```diff
@@ -55,6 +58,19 @@ class _SetupBedtimeScreenState extends State<SetupBedtimeScreen> {
       canPop: false,
       child: BaseStateConsumer<dynamic, Bedtime>(
         cubit: _cubit,
+        onError: (context, message) => ColoredBox(
```

Comment:
I don't see the error state being emitted in this PR. Was that already in the current cubit?

Reply:
yes it was already part of the cubit, but not handled in the UI, that was my bad
`  Future<void> onClickContinue(
    String guid,
    double bedtimeSliderValue,
    int windDownValue,
  ) async {
    emitLoading();
    final bedtime = Bedtime(
      id: guid,
      bedtimeInUtc: _bedtimeSliderValueToUtc(bedtimeSliderValue),
      winddownMinutes: windDownValue,
    );
    try {
      await _editChildRepository.editChildBedtime(bedtime);
      emitCustom(bedtime);
    } catch (e, s) {
      LoggingInfo.instance.logExceptionForDebug(e, stacktrace: s);
      emitError('An unexpected error occurred while saving bedtime settings.');
    }
  }`